pages/pricing.js

```diff
@@ -63,7 +63,7 @@ export default function Premium({ user }) {
           }
           return false;
         },
-        onClick: () => {},
+        onClick: () => { },
```

Comment:
Please remove the space in between those curly braces and keep it as it was before.